test/extension/utils.test.js

```diff
@@ -68,47 +71,43 @@ describe('Test extension utils', () => {
     };
   });
 
+  afterEach(() => {
+    sandbox.restore();
+  });
+
   it('log', async () => {
-    const spy = sinon.spy(console, 'log');
-    window.LOG_LEVEL = 4;
+    const spy = sandbox.spy(console, 'log');
     utils.log.error('foo');
     expect(spy.calledWith('ERROR', 'foo')).to.be.true;
     utils.log.warn('foo');
     expect(spy.calledWith('WARN', 'foo')).to.be.true;
-    utils.log.info('foo');
-    expect(spy.calledWith('INFO', 'foo')).to.be.true;
-    utils.log.debug('foo');
-    expect(spy.calledWith('DEBUG', 'foo')).to.be.true;
-    delete window.LOG_LEVEL;
-    spy.restore();
   });
 
   it('i18n', async () => {
-    const spy = sinon.spy(window.chrome.i18n, 'getMessage');
+    const spy = sandbox.spy(window.chrome.i18n, 'getMessage');
     // simple call
     utils.i18n('hello');
     expect(spy.calledWith('hello')).to.be.true;
     // call with subs
     utils.i18n('hello $1', ['world']);
     expect(spy.calledWith('hello $1', ['world'])).to.be.true;
-    spy.restore();
   });
 
   it('url', async () => {
-    const spy = sinon.spy(window.chrome.runtime, 'getURL');
+    const spy = sandbox.spy(window.chrome.runtime, 'getURL');
     utils.url('/foo');
     expect(spy.calledWith('/foo')).to.be.true;
-    spy.restore();
   });
 
   it('checkLastError', async () => {
+    chrome.runtime.lastError = new Error('foo');
     const lastError = utils.checkLastError();
     expect(lastError).to.exist;
-    expect(lastError.message).to.equal('foo');
+    expect(lastError).to.equal('foo');
+    chrome.runtime.lastError = null;
```

Comment:
maybe also move to `afterEach()` ?

Reply:
`chromeMock` does not acively set it, so I think it's safe to keep in this test.